flopy/utils/get_modflow.py

```diff
@@ -179,8 +181,15 @@ def run_main(
     else:
         req_url = f"{api_url}/releases/tags/{release_id}"
     try:
-        with urllib.request.urlopen(req_url) as resp:
+        hdr = {"Authorization": f"Bearer {token}"} if token else {}
```

Comment:
nice catch! I see this documented [here](https://docs.github.com/en/actions/security-guides/automatic-token-authentication). However, I might change the approach to use `os.environ.get("GITHUB_TOKEN")` internally rather than taking a `--token` argument. I can handle this later, however, so keep this approach for now.

Reply:
Ok, sounds good- that does sound better. I guess end users aren't likely to hit rate limits, only CI.

Reply:
If your PR is merged before this one I'll rebase to get the changes you've made in #1473 and use that approach in the CI configs.